frappe/core/doctype/user/user.py

```diff
@@ -240,9 +240,9 @@ def send_welcome_mail_to_user(self):
 
 		link = self.reset_password()
 
-		app_title = [t for t in frappe.get_hooks('app_title') if t != 'Frappe Framework']
+		app_title = [t for t in frappe.get_hooks('app_title') if t == 'ERPNext']
```

Comment:
I agree this design needs to be changed.

ideally there should not no mention of 'erpnext' in frappe.

how about a simple hook like `get_site_title` that will return the company name if erpnext is installed or just the app name?